browser/BrowserPerf.proto

```diff
@@ -33,10 +33,14 @@ message BrowserPerfData {
     int32 serviceId = 1;
     // Service version in browser is the Instance concept in the backend.
     int32 serviceVersionId = 2;
+    // Data time
+    int64 time = 3; // assignment by the backend
```

Comment:
Please don't add `assignment by the backend`. This could be set by the client side, then backend doesn't need to set it

Reply:
deleted